.github/PR_Template.md

```diff
@@ -0,0 +1,20 @@
+Thank you for submitting this PR!
+
+**Please check if the PR fulfills these requirements**
+- [ ] The commit message follows our guidelines: `trans(title): topic1, topic2 or review(title): topic1, topic2`
+- [ ] The translation content follows our commit conventions
+
+**This PR is solving a issue? Please provide the issue number (ex: issue #29)**
```

Comment:
`Is the PR solving an issue? Please provide the number`

We should also make sure to include in the footer of the commit messages the `Closes #[number]`